src/signals/settings/components/PageHeader/index.js

```diff
@@ -20,6 +20,7 @@ const StyledSection = styled.section`
 const StyledHeading = styled(Heading)`
   margin: 0;
   line-height: 44px;
+  font-weight: 400;
```

Comment:
There are several changes in this PR like the one on this line. This seems not to be in scope of this PR and I'm quite certain that those changes are already present in `develop` and `master`. Have you brought your branch up to date with `develop`?

Reply:
solved by updating the base branch with `develop`